docs/how-to-guides/how-to-integrate-autoware-with-diff-drive-vehicle.md

```diff
@@ -0,0 +1,59 @@
+# How to integrate Autoware with differential drive vehicle
+
+## 1. Integrate Autoware with differential drive vehicle
+
+Currently, Autoware assumes the vehicle to follow an Ackermann kinematic model that uses Ackermann steering.
+Thus, the Autoware interfaces adopt the Ackermann commands format for the output of the control module (see [the definition of AckermannControlCommands.idl](https://gitlab.com/autowarefoundation/autoware.auto/autoware_auto_msgs/-/blob/master/autoware_auto_control_msgs/msg/AckermannControlCommand.idl) for details).
```

Comment:
I agree that it's better to add the ROS definition, thanks!

I thought that adding both the ROS definition and the link to AckermannLateralCommand will make it too long, so how about something like this? https://github.com/autowarefoundation/autoware-documentation/pull/125/commits/c108bf19388e60d83fa9b75335a897344649cb5f

IMO the link to AckermannLateralCommand is not necessary since AckermannControlCommand obviously calls that message type.

Reply:
Sorry I missed this one.
Reflected the feedback here: https://github.com/autowarefoundation/autoware-documentation/pull/125/commits/834eb30decbcf1074f2657458ae5fc3a79d4a3f0